core/interfaces.go

```diff
@@ -7,9 +7,11 @@ package core
 
 import (
 	"crypto/x509"
-	jose "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/square/go-jose"
 	"net/http"
 	"time"
+
+	jose "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/square/go-jose"
```

Comment:
It is; it was not in the preferred import order of "system", "vendorized", "internal".